apps/web-client/src/routes/orders/suppliers/reconcile/[id]/+page.ts

```diff
@@ -0,0 +1,47 @@
+import { getReconciliationOrder, getReconciliationOrderLines } from "$lib/db/cr-sqlite/order-reconciliation";
+import type { PageLoad } from "./$types";
+import { getPlacedSupplierOrderLinesForReconciliation } from "$lib/db/cr-sqlite/suppliers";
+import type { BookEntry } from "@librocco/db";
+
+export const load: PageLoad = async ({ parent, params, depends }) => {
+	depends("reconciliationOrder:data");
+
+	const { ordersDb } = await parent();
+	const reconciliationOrder = await getReconciliationOrder(ordersDb, parseInt(params.id));
+	const reconciliationOrderLines = await getReconciliationOrderLines(ordersDb, parseInt(params.id));
+
+	const supplierOrderIds = JSON.parse(reconciliationOrder.supplier_order_ids) || [];
+
+	const placedOrderLines = await getPlacedSupplierOrderLinesForReconciliation(ordersDb, supplierOrderIds);
+
+	const supplierOrders = placedOrderLines.reduce<{ [supplier_order_id: string]: { supplier_name: string; supplier_id: string } }>(
+		(acc, val) =>
+			acc[val.supplier_order_id]
+				? acc
+				: { ...acc, [val.supplier_order_id]: { supplier_name: val.supplier_name, supplier_id: val.supplier_id } },
+		{}
+	);
+
+	const books = reconciliationOrderLines.reduce(
```

Comment:
`getReconciliationOrderLines` should get the book data, similar to how we get supplier order lines